bench/perf_run_py/perf_stat.py

```diff
@@ -133,7 +133,27 @@ def get_value(self, nth_max: int):
 
     def get_value_list(self, brief: bool):
         as_list = []
-        for i in range(1, self.num_max + 1):
+        num_max = len(self.values) if self.num_max > len(self.values) else self.num_max
+        for i in range(1, num_max + 1):
+            as_dict = {
+                'name': self.output_label + str(i),
+                'value': self.get_value(i)
+            }
+            if not brief:
+                as_dict['values'] = self.values
```

Comment:
No, currently we do not have a ticket to investigate a fix. I do think it would be good to only print all the latencies once. Created [WT-8438](https://jira.mongodb.org/browse/WT-8438)